.github/workflows/ci.yml

```diff
@@ -0,0 +1,88 @@
+name: go-workflow
+on:
+  push:
+    # Temporarily removed for first PR
+    # branches:
+    #   - master
+  pull_request:
+jobs:
+  golang-tests:
+    name: tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        go-version: ["1.15", "1.14", "1.13"]
+        go111module: ["on", "off"]
+        include:
+          # includes goflags when go111module is on
+          - go111module: "on"
+            goflags: -mod=readonly
```

Comment:
This block adds `goflags` *only* for the `go111module: on` configuration.

Reply:
Would you be able to verify via the logs that `goflags` is being used properly for module mode on?

Reply:
Hard to tell with the current output. We can debug it with `go env` to print Go-related variables.

Reply:
If I understood https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions#using-environment-variables-in-a-matrix well, we need to set the env explicitly in the step(s) that we need it.

`go env` will tell us if we got it right.